README.md

```diff
@@ -189,90 +189,7 @@ There are several custom markdown plugins that are available by default that enh
-
-<Tabs>
-<Tab heading="CLI command">
-            <!-- Intentionally skipped line.. -->
-```shell-session
-$ command ...
-```
-            <!-- Intentionally skipped line.. -->
-</Tab>
-<Tab heading="API call using cURL">
-
-```shell-session
-$ curl ...
-```
-
-</Tab>
-</Tabs>
-
-Continued normal markdown content
-````
-
-The intentionally skipped line is a limitation of the mdx parser which is being actively worked on. All tabs must have a heading, and there is no limit to the number of tabs, though it is recommended to go for a maximum of three or four.
-
-#### Enterprise Alert
-
-This component provides a standard way to call out functionality as being present only in the enterprise version of the software. It can be presented in two contexts, inline or standalone. Here's an example of standalone usage from the Consul docs website:
-
-![Enterprise Alert Component - Standalone](https://p176.p0.n0.cdn.getcloudapp.com/items/WnubALp8/Screen%20Shot%202020-06-11%20at%206.06.03%20PM.png?v=d1505b90bdcbde6ed664831a885ea5fb)
-
-The standalone component can be used as such in markdown files:
-
-```mdx
-# Page Headline
-
-<EnterpriseAlert />
-
-Continued markdown content...
-```
-
-It can also receive custom text contents if you need to change the messaging but wish to retain the style. This will replace the text `This feature is available in all versions of Consul Enterprise.` with whatever you add. For example:
-
-```mdx
-# Page Headline
-
-<EnterpriseAlert>
-  My custom text here, and <a href="#">a link</a>!
-</EnterpriseAlert>
-
-Continued markdown content...
-```
-
-It's important to note that once you are adding custom content, it must be html and can not be markdown, as demonstrated above with the link.
-
-Now let's look at inline usage, here's an example:
-
-![Enterprise Alert Component - Inline](https://p176.p0.n0.cdn.getcloudapp.com/items/L1upYLEJ/Screen%20Shot%202020-06-11%20at%206.07.50%20PM.png?v=013ba439263de8292befbc851d31dd78)
-
-And here's how it could be used in your markdown document:
-
-```mdx
-### Some Enterprise Feature <EnterpriseAlert inline />
-
-Continued markdown content...
-```
-
-It's also worth noting that this component will automatically adjust to the correct product colors depending on the context.
-
-#### Other Components
-
-Other custom components can be made available on a per-site basis, the above are the standards. If you have questions about custom components that are not documented here, or have a request for a new custom component, please reach out to @hashicorp/digital-marketing.
+A number of custom [mdx components](https://mdxjs.com/) are available for use within any `.mdx` file. Please refer to the [Swingset](https://react-components.vercel.app/?component=Tabs) documentation for the latest examples and API reference. If you have questions about custom components that are not documented here, or have a request for a new custom component, please reach out to @hashicorp/digital-marketing.
```

Comment:
Only a subset of those components are available by default (basically `Tabs`, `EnterpriseAlert` and code blocks); all the remaining ones need to be imported in the doc file manually. Since that's the case I'd prefer we not link out to the component API at all.

Reply:
I'm ok with this if you are Laura :)

Reply:
yes! I will remove :)